merlin/examples/workflows/openfoam_wf_singularity/requirements.txt

```diff
@@ -1,3 +1,3 @@
 Ofpp==0.11
-scikit-learn==0.21.3
```

Comment:
This can just be >=0.21.3 to capture the old behavior.

Reply:
Does this need to be changed in the other openfoam examples as well?

Reply:
I think so. I just tried this again and it seems to work fine now. Not sure why/how I was seeing the strange behavior I mentioned below... I'll get the requirements files changed so we can get this merged.

Reply:
> This can just be >=0.21.3 to capture the old behavior.

@koning I've been thinking about this more and since the setup error was happening in v0.21.3 I'm not sure if we should keep the = sign here in the requirements file. In other words I think the version has to be newer than 0.21.3 in order to avoid the setup error we were seeing

Reply:
OK, just trying to keep the other installs consistent. If the setup is the issue, then it should be updated.